src/app/core/services/models/rundown-entity.service.ts

```diff
@@ -163,6 +163,10 @@ export class RundownEntityService {
   }
 
   public insertPartInSegment(rundown: Rundown, part: Part): Rundown {
+    if (part.isNext) {
+      return this.insertPartAsNext(rundown, part)
```

Comment:
This inserts the Part into the Segment and on line 177 the Part is  inserted into the Segment again, meaning the Part will be inserted twice???

Reply:
Nevermind, I just noticed the `return` :p